src/components/shared/Entity/Create/Config.tsx

```diff
@@ -0,0 +1,114 @@
+import { Box, CircularProgress, Collapse, Typography } from '@mui/material';
+import ConnectorTiles from 'components/ConnectorTiles';
+import ExistingEntityCards from 'components/shared/Entity/ExistingEntityCards';
+import {
+    useExistingEntity_connectorName,
+    useExistingEntity_createNewTask,
+    useExistingEntity_hydrated,
+} from 'components/shared/Entity/ExistingEntityCards/Store/hooks';
+import useEntityCreateNavigate from 'components/shared/Entity/hooks/useEntityCreateNavigate';
+import useGlobalSearchParams, {
+    GlobalSearchParams,
+} from 'hooks/searchParams/useGlobalSearchParams';
+import useBrowserTitle from 'hooks/useBrowserTitle';
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { EntityWithCreateWorkflow } from 'types';
+
+interface Props {
+    title: string;
+    entityType: EntityWithCreateWorkflow;
+}
+
+function EntityCreateConfig({ title, entityType }: Props) {
+    useBrowserTitle(title);
+
+    const [connectorId] = useGlobalSearchParams([
+        GlobalSearchParams.CONNECTOR_ID,
+    ]);
```

Comment:
This does not have to be an array - the function will take a single param and return a single item.

Reply:
Yes, I know that. This code was shuffled quite a bit over the lifetime of this branch and an additional search parameter was extracted in legacy code. Will cleanup.